### ISSUE TITLE
still maintained?

### ISSUE THREAD

**Issue by kberry** [pkubowicz/opendetex#79]
Are you still actively maintaining the code here? No commits since 2020, 40+ open issues. If you're stepping down, please let me know and we will maintain it in TeX Live again (unless and until another volunteer comes along). Thanks. --karl (at) freefriends (dot) org

**Comment by pauloney** [comment]
Peter, how is your time? I can move Yoni here and deal with some of the trivial stuff first till he ramps up on the program. Would that be any good? I do not want to accumulate a huge number of patches, if you are busy with other stuff.

Paulo Ney

**Comment by kberry** [comment]
Thanks for restarting this ... FWIW, I just submitted a patch (issue 82) for an unterminated \verb command, which causes a crash. I hope someone does something with it, and makes a new release. Meanwhile, I committed it to TeX Live (r64408), along with a variety of minor things I noticed along the way, which I submitted here as separate issues.

I note there are quite a number of bug reports. The two pending PR's do not fix them :(. Happy hacking :).
